rocAL/include/api/rocal_api_types.h

```diff
@@ -259,7 +259,10 @@ enum RocalDecoderType {
     ROCAL_DECODER_VIDEO_FFMPEG_SW = 3,
     /*! \brief AMD ROCAL_DECODER_VIDEO_FFMPEG_HW
      */
-    ROCAL_DECODER_VIDEO_FFMPEG_HW = 4
+    ROCAL_DECODER_VIDEO_FFMPEG_HW = 4,
+    /*! \brief AMD ROCAL_DECODER_AUDIO_SNDFILE
+     */
+    ROCAL_DECODER_AUDIO_SNDFILE = 5
```

Comment:
I would suggest calling it ROCAL_DECODER_AUDIO_GENERIC instead.  We can mention that it uses libSndFile for implemetation